### PR TITLE
[DEV APPROVED] 10468 footer changes

### PR DESCRIPTION
**Pull request by andreferreiradlw**
**Target Process ticket**

[TP 10468](https://moneyadviceservice.tpondemand.com/RestUI/Board.aspx#page=board/5682182231901749200&appConfig=eyJhY2lkIjoiNTM0ODcyOThDQjBDMTMyMDgyMTBENEZEM0QwM0ZBQzkifQ==&boardPopup=userstory/10468/silent)

**Summary**

Changes made in the secondary footer and copyright message in order to match it with the footer in blog. Some general styles shared between the two footers were moved to yeast in [PR 52](https://github.com/moneyadviceservice/yeast/pull/52).

### CODE REVIEW

**Comment by davidtrussler** [comment]
Do we still need the image marked "(old)"? I'm guessing this could be deleted.

File: app/views/shared/_footer_secondary.html.erb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <div class="l-footer-secondary t-footer-secondary">
-  <div class="footer-secondary--inner">
+  <div class="l-constrained">
+  <div class="l-footer-secondary--inner">
```
Comment (davidtrussler) [inline app/views/shared/_footer_secondary.html.erb]: Minor thing: could you indent this?

Comment (andreferreiradlw) [reply app/views/shared/_footer_secondary.html.erb]: I will be indenting the entire document just before I merge this PR.

File: app/assets/stylesheets/components/common/_footer_secondary.scss [hunk 2]
```diff
@@ -102,37 +35,4 @@
...
-    align-items: center;
-    -ms-flex-pack: justify;
-    justify-content: space-between;
-  }
-}
-
-.footer-secondary__list--right {
-  max-width: 50%;
-}
+}
```
Comment (margOnline) [inline app/assets/stylesheets/components/common/_footer_secondary.scss]: new line

Comment (andreferreiradlw) [reply app/assets/stylesheets/components/common/_footer_secondary.scss]: Added now

**Pull request merged**